package.json

```diff
@@ -7,7 +7,8 @@
     "build": "npm run lint && webpack --config ./webpack.config.prod.js -p",
     "develop": "webpack-dev-server --open --config ./webpack.config.develop.js",
     "lint": "eslint *.js",
-    "serve": "npm run build && http-server ./build -o -p 9000"
+    "serve": "npm run build && http-server ./build -o -p 9000",
+    "test": "jest --coverage"
```

Comment:
We should add a test step here to [_config.yml_](https://github.com/ProvidenceGeeks/website-frontend/blob/master/.circleci/config.yml#L30) to make sure tests get run on every build